test/Api.Test/SecretsManager/Controllers/RequestSMAccessControllerTests.cs

```diff
@@ -0,0 +1,73 @@
+﻿#nullable enable
+using System.Security.Claims;
+using Bit.Api.SecretsManager.Controllers;
+using Bit.Api.SecretsManager.Models.Request;
+using Bit.Core.AdminConsole.Entities;
+using Bit.Core.Entities;
+using Bit.Core.Exceptions;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Bit.Test.Common.AutoFixture;
+using Bit.Test.Common.AutoFixture.Attributes;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Api.Test.SecretsManager.Controllers;
+
+[ControllerCustomize(typeof(RequestSMAccessController))]
+[SutProviderCustomize]
+public class RequestSMAccessControllerTests
+{
+    [Theory]
+    [BitAutoData]
+    public async Task RequestSMAccessFromAdminst_WhenSendingNoModel_ShouldThrowNotFoundException(
+    User user, SutProvider<RequestSMAccessController> sutProvider)
+    {
+        // Arrange
+        sutProvider.GetDependency<IUserService>().GetUserByPrincipalAsync(Arg.Any<ClaimsPrincipal>()).Returns(Task.FromResult(user));
+        sutProvider.GetDependency<IOrganizationRepository>().GetByIdentifierAsync(Arg.Any<string>()).ReturnsForAnyArgs((Organization)null);
+
+        // Act & Assert
+        await Assert.ThrowsAsync<NotFoundException>(() => sutProvider.Sut.RequestSMAccessFromAdmins(new RequestSMAccessRequestModel()));
+    }
+
+    [Theory]
+    [BitAutoData]
+    public async Task RequestSMAccessFromAdminst_WhenSendingValidData_ShouldSucceed(
+    User user,
+    RequestSMAccessRequestModel model,
+    Core.AdminConsole.Entities.Organization org,
+    SutProvider<RequestSMAccessController> sutProvider)
+    {
+        // Arrange
+        sutProvider.GetDependency<IOrganizationRepository>().GetByIdentifierAsync(Arg.Any<string>()).ReturnsForAnyArgs(Task.FromResult(org));
+        sutProvider.GetDependency<IUserService>().GetUserByPrincipalAsync(Arg.Any<ClaimsPrincipal>()).Returns(Task.FromResult(user));
+        model.OrganizationId = org.Id;
+
+        // Act & Assert
+        await sutProvider.Sut.RequestSMAccessFromAdmins(model);
```

Comment:
can you add an assert / validation on the Command, so it's not just 200, but also an assert on the arguments for command ?